Documentation/raw data (#12)
qbicsoftware/research-data-management: docs/rawdata/raw_data_download_run_download.md

# Download Raw Data

We show how to run the download via the two popular command line clients [cURL](https://curl.se/docs/manpage.html) and [GNU Wget](https://www.gnu.org/software/wget/).
Of course, you can use any other software that supports HTTP.

Keep in mind that you need to be [granted access](../project/project_access.md)
by the project owner to download the associated measurement data.

The commands contain two placeholders, where you have to provide your custom input:

- *ACCESS_TOKEN*: your generated [personal access token](raw_data_download_create_pat.md)

> - *MEASUREMENT_URL*: the [URL](raw_data_download_acquire_urls.md) of the to be downloaded measurement

Reviewer: ```suggestion
- *MEASUREMENT_URL*: the [URL](raw_data_download_acquire_urls.md) associated with the data of the measurements
```

=== "curl"
    
    ``` bash
    curl -OJ -H "Authorization: Bearer <ACCESS_TOKEN>" <MEASUREMENT_URL>
    ```

=== "wget"

    ``` bash
    wget --content-disposition --trust-server-names --header "Authorization: Bearer <ACCESS_TOKEN>" <MEASUREMENT_URL>
    ```

> You can download multiple measurements. All you have to do is provide the URLs in separate lines.

Reviewer: ```suggestion
To download multiple measurements using one command, you can provide a file containing one measurement URL per line (as the one you can download in the Data Manager).
```
